src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/WorkspaceSemanticTokensRefreshNotifier.cs

```diff
@@ -73,6 +73,12 @@ public void NotifyWorkspaceSemanticTokensRefresh()
                 return;
             }
 
+            // We could have been called before the LSP server has even been initialized
+            if (!_clientCapabilitiesService.CanGetClientCapabilities)
```

Comment:
I can believe it, but do we know what is actually calling this before the server has initialized?

Reply:
With only 3 hits, there is a big chance this never happens, but it seemed like a cheap fix. Maybe it was a small window of time between dispatcher changes and startup services changes.